lib/client/src/snapshot_source.rs

```diff
@@ -114,7 +128,7 @@ async fn feed_snapshots(
     let mut snapshot = AccountSnapshot::default();
 
     // Get all accounts of the mango program
-    let response = rpc_client
+    let response = rpc_client_scan
```

Comment:
@ckamm  any idea if I can avoid having two clients?

Reply:
Not that I'm aware of; but two http connections are also acceptable.

Reply:
commented